projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -83,7 +83,12 @@ export class NavigationListComponent {
   }
 
   public navigate(item: NavItemLinkConfig): void {
-    this.navigationService.navigateWithinApp(item.matchPaths[0], this.activatedRoute);
+    this.navigationService.navigate({
+      navType: NavigationParamsType.InApp,
+      path: item.matchPaths[0],
+      relativeTo: this.activatedRoute,
+      replaceCurrentHistory: item.skipLocationChange
```

Comment:
So missed this the first time around, but we should clarify our naming (the test actually caught this). There's two separate booleans that the angular supports:
- `skipLocationChange`: When true, navigates without pushing a new state into history.
- `replaceUrl`: When true, navigates while replacing the current state in history.

To illustrate the distinction, imagine three nav items that are navigated in order (`first -> second -> third`)
```
first: { ...,  skipLocationChange: false},
second: { ...,  skipLocationChange: true},
third: { ...,  skipLocationChange: false}
```

with the behavior of `skipLocationChange`, when complete our history stack looks like:
```
<third> // Active
<first> // browser back
```

with the behavior of `replaceUrl`, when complete our history stack looks like:
```
<third> // Active
<second> // browser back
```



A subtle distinction, but our implementation uses `replaceUrl`, so we should name the item's property accordingly.

Reply:
@DmitiryPotychkin can we update the naming here to reflect the behavior? In other words, `skipLocationChange` on line 124 should be named `replaceCurrentHistory`.

Reply:
@aaron-steinfeld thank you for the explanation!